infra/templates.py

```diff
@@ -47,7 +47,7 @@
 """
 
 EXTERNAL_DOCKER_TEMPLATE = """\
-FROM gcr.io/oss-fuzz-base/%(base_builder)s
+FROM gcr.io/oss-fuzz-base/%(base_builder)s:v1
```

Comment:
This applies to OSS-Fuzz projects as well, but I think that's a good thing to be more explicit going forward. It's not something that changes very often either.